repo2docker/buildpacks/r.py

```diff
@@ -64,21 +71,22 @@ def detect(self):
         unless a `requirements.txt` is present and we do not want to require the
         presence of a `requirements.txt` to use R.
 
-        Instead we just check if runtime.txt contains a string of the form
-        `r-<YYYY>-<MM>-<DD>`
+        Instead we check the options described in the class comment above.
```

Comment:
I'd delete this line if we don't want to duplicate the contents of the doc string for the class. Ideally we'd be able to come up with a nice short summary about all the various ways to trigger the R buildpack but I currently can't think of one :-(

Reply:
Deleted. Your call, really. I guess the "nice short summary" means it can't be automated.

Do you think there should be a developer-facing variant of the config files docs?

Reply:
Some developer facing docs would be good. Maybe building on https://repo2docker.readthedocs.io/en/latest/architecture.html?